routes/redirect.js

```diff
@@ -104,6 +104,7 @@ module.exports = function (app) {
     route: 'https://teach.mozilla.org/teach-like-mozilla/web-literacy',
     paths: [
       '/resources/*',
```

Comment:
We probably want to add a redirect for `/resources` as well. Looks like the current implementation is actually redirecting twice:

```bash
curl --head https://webmaker.org/resources
```

```bash
HTTP/1.1 303 See Other
Date: Tue, 26 May 2015 16:57:35 GMT
Location: /resources/
Strict-Transport-Security: max-age=15768000
X-Content-Type-Options: nosniff
X-FRAME-OPTIONS: ALLOW-FROM http://optimizely.com
X-XSS-Protection: 1; mode=block
Connection: keep-alive
```